src/main/ImGuiManager.cpp

```diff
@@ -1203,6 +1203,18 @@ void ImGuiManager_OverlaySettings()
 	mq::imgui::HelpMarker("When enabled, MacroQuest will take over drawing the EQ Cursor when\n"
 		"it is near to or hovering over an ImGui window.");
 
+	if (ImGui::Checkbox("Ignore Window Clamping", &s_imguiIgnoreClampWindow))
+	{
+		WritePrivateProfileBool("Overlay", "ImGuiIgnoreClampWindow", s_imguiIgnoreClampWindow, mq::internal_paths::MQini);
+
+		auto& io = ImGui::GetIO();
+		if (io.ImGuiIgnoreClampWindow != s_imguiIgnoreClampWindow)
```

Comment:
the condition is unnecessary

Reply:
You also need to set this when the context is created

Reply:
fixed.